package.json

```diff
@@ -18,12 +20,20 @@
   },
   "homepage": "https://github.com/Front-End-Ninjas/image-gallery#readme",
   "dependencies": {
+    "axios": "^0.18.0",
     "express": "^4.16.2",
     "path": "^0.12.7",
     "pg": "^7.4.1",
-    "pg-format": "^1.0.4"
+    "pg-format": "^1.0.4",
```

Comment:
Not sure if this dependency should be in dev since its seeding data. It's making me think about my situation as well lol